Queueomatic/Client/Components/Login/LoginComponent.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Net.Http.Json;
+using System.Text.Json;
+using Microsoft.AspNetCore.Components;
+using Queueomatic.Shared.DTOs;
+
+namespace Queueomatic.Client.Components.Login;
+
+public partial class LoginComponent : ComponentBase
```

Comment:
This cs file should use code isolation like the css file uses.

Reply:
What do you mean by this?

Reply:
Oh, I see now. It is wrong, I will fix it.